### PR TITLE
elsearticle:0.1.0

### PR DESCRIPTION
**Pull request by maucejo**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: `elsearticle` is a Typst template that aims to mimic the Elsevier article LaTeX class, a.k.a. `elsearticle.cls`, provided by Elsevier to format manuscript properly for submission to their journals.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/elsearticle/0.1.0/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Pg Biel
```
Comment (elegaanz) [inline packages/preview/elsearticle/0.1.0/LICENSE:3]: You probably want to change that for your name.

File: packages/preview/elsearticle/0.1.0/template/main.pdf
Comment (elegaanz) [inline packages/preview/elsearticle/0.1.0/template/main.pdf:1]: Can you please delete this file?

File: packages/preview/elsearticle/0.1.0/template/main.typ [hunk 2]
```diff
@@ -0,0 +1,154 @@
+#import "../src/elsearticle.typ": *
```
Comment (elegaanz) [inline packages/preview/elsearticle/0.1.0/template/main.typ:1]: This import should not be relative.

```suggestion
#import "@preview/elsearticle:0.1.0": *
```

File: packages/preview/elsearticle/0.1.0/template/main.pdf
Comment (maucejo) [reply packages/preview/elsearticle/0.1.0/template/main.pdf:1]: I have removed `main.pdf`.

**Pull request merged**